### ISSUE TITLE
Surveys

### ISSUE THREAD

**Issue by oriolgual** [HospitaletDeLlobregat/decidim-hospitalet#13]
#### :tophat: User Story

TLDR: We need to make an online version of this: https://drive.google.com/file/d/0B2voJ3vHaOfJY0tCUFlreTl4RHM/view

This will be a component only available for Hospitalet (we might generalize it un the future and add it to decidim).

This component will add a feature to processes so that citizens can answer a predefined survey (no surveys can be created at the admin).

Take a look at other components in order to get started.

##### Properties

- Scope (mandatory, select, use scopes to populate the options)
- Categories (mandatory, multiple checkboxes up to 4 options, use categories to populate the options)
- Other priorities (optional, textarea)
- Future ideas (optional, textarea)
- Gender (optional, select, female, male , other)
- Age group (optional, select, 18-24; 25-34;35-44,45-54;55+)
- Zip code (optional, numeric, 5 digits)
- Living at scope? (optional, radio button, yes/no)
- Working at scope? (optional, radio button, yes/no)
- City (optional, select, Hospitalet de Llobregat, Other)
- Name (optional, text)
- Phone (optional, text)

##### Considerations

* Only one submitted survey per user
* Only signed-up users can submit a survey
* After submitting the survey, users should be encouraged to send a proposal
* We don't need an admin interface, it will be specified in another PR.
* The survey results aren't publicly available
* Users can't change their survey submission

##### Implementation details

* Try to KISS but keeping in mind that we'll later generalise this. Maybe we can save all the answers in a single JSON column.

#### :clipboard: Related documentation

Further fields description:
https://docs.google.com/document/d/19enaJK750wC8VIijhoc9tsdJX32fCdNp1_EpGY0X-K0/edit

Example form to use as styles:
http://decidim-design.herokuapp.com/new-proposal/

#### :dart: Acceptance criteria

As a user I can send one submission of the survey. 

#### :pushpin: Related issues

- #11

**Comment by carolromero** [comment]
@oriolgual I've updated all the field values for the survey form.

Catalan version:
https://docs.google.com/document/d/19enaJK750wC8VIijhoc9tsdJX32fCdNp1_EpGY0X-K0/edit?usp=sharing

Spanish version:
https://drive.google.com/open?id=1vvaXOM-tyFKU0X24c592ZRGmXKPaJi50skVGJp9vJLc
